packages/net-stubbing/lib/external-types.ts

```diff
@@ -367,6 +397,49 @@ declare global {
        * @deprecated
        */
       route2(method: Method, url: RouteMatcher, response?: RouteHandler): Chainable<null>
+      /**
+       * Wait for a specific request to complete.
+       *
+       * @see https://on.cypress.io/wait
+       * @param {string} alias - Name of the alias to wait for.
+       *
+      ```
+      // Wait for the route aliased as 'getAccount' to respond
+      // without changing or stubbing its response
+      cy.http('https://api.example.com/accounts/*').as('getAccount')
+      cy.visit('/accounts/123')
+      cy.wait('@getAccount').then((xhr) => {
+        // we can now access the low level request
+        // that contains the request body,
+        // response body, status, etc
+      })
+      ```
+      */
+      wait(alias: string, options?: Partial<WaitOptions>): Chainable<Request>
+      /**
+       * Wait for list of requests to complete.
+       *
+       * @see https://on.cypress.io/wait
+       * @param {string[]} aliases - An array of aliased routes as defined using the `.as()` command.
+       *
+      ```
+      // wait for 3 XHR requests to complete
+      cy.server()
+      cy.route('users/*').as('getUsers')
+      cy.route('activities/*').as('getActivities')
+      cy.route('comments/*').as('getComments')
```

Comment:
Shouldn't this description show `cy.http()` examples?
```suggestion
      cy.http('users/*').as('getUsers')
      cy.http('activities/*').as('getActivities')
      cy.http('comments/*').as('getComments')
```

Reply:
good catch